ruma-events-macros/src/event_content.rs

```diff
@@ -61,7 +59,7 @@ impl Parse for EventMeta {
 }
 
 /// Create an `EventContent` implementation for a struct.
-pub fn expand_event_content(input: &DeriveInput) -> syn::Result<TokenStream> {
+pub fn expand_event_content(input: &DeriveInput, emit_redacted: bool) -> syn::Result<TokenStream> {
```

Comment:
This is actually the only function that needs this parameter. Every other `_expand_event_content` is only ever called with `true`, or `false`.